### PR TITLE
React appbar

### PR DESCRIPTION
**Pull request by Octopus773**
adding nav bar
and fetch on save

### CODE REVIEW

File: web-app/src/pages/Login/LoginPage.tsx [hunk 1]
```diff
@@ -59,6 +63,24 @@ type AuthCompProps = {
 	isConfirmButtonVisible: boolean;
 };
 
+const requestLogin = async (username: string, password: string, signup: boolean): Promise<boolean> => {
+	const rawResponse = await fetch( API_ROUTE + '/auth/' + (signup ? "signup" : "login"), {
+	  method: 'POST',
```
Comment (AnonymusRaccoon) [inline web-app/src/pages/Login/LoginPage.tsx:68]: The indentation is broken

File: web-app/src/pages/Login/LoginPage.tsx [hunk 2]
```diff
@@ -84,9 +106,10 @@ export default function AuthComponent() {
 		}));
 	}, [authData.username, authData.password, authData.confirmedPassword]);
 
-	const handleLogin = () => {
-		//TODO Implements back auth routes
-		if (authData.username === "b" && authData.password === "b") {
+	const handleLogin = async () => {
+
```
Comment (AnonymusRaccoon) [inline web-app/src/pages/Login/LoginPage.tsx:110]: Why is there 2 empty lines?

File: web-app/src/pages/PipelineEdit/PipelineEditParams.tsx [hunk 3]
```diff
@@ -17,26 +23,32 @@ export default function PipelineNameSetup({ title }: PipelineActionListProps) {
...
+
+				}
+				
 				<LoadingButton
 					sx={{ marginTop: "30px" }}
 					color="secondary"
 					loading={false}
 					loadingPosition="start"
 					startIcon={<Save />}
+					onClick={() => setParams({
```
Comment (AnonymusRaccoon) [inline web-app/src/pages/PipelineEdit/PipelineEditParams.tsx:50]: The coding style is broken.

File: web-app/src/pages/PipelineEdit/PipelineEditPipeline.tsx [hunk 4]
```diff
@@ -0,0 +1,153 @@
...
+
+	return (
+		<div>
+			<div
+				style={{
+					display: "grid",
+					gridTemplateColumns: "25vw 5vw 12vw 13vw",
+					gridTemplateRows: "2fr 1fr 40vh 3fr 1fr",
+					gridTemplateAreas: `
+							'pipelineTitle 	pipelineTitle	pipelineTitle		enabledStatus'
```
Comment (AnonymusRaccoon) [inline web-app/src/pages/PipelineEdit/PipelineEditPipeline.tsx:48]: Please allign with spaces.

File: web-app/src/pages/Login/LoginPage.tsx (hunk 1 above)
Comment (0Nom4D) [reply web-app/src/pages/Login/LoginPage.tsx]: ## Fix

Identation is fixed with Prettier.

File: web-app/src/pages/Login/LoginPage.tsx (hunk 2 above)
Comment (0Nom4D) [reply web-app/src/pages/Login/LoginPage.tsx]: ## Fix

Identation is fixed with Prettier.

File: web-app/src/utils/CRUDPipeline.tsx [hunk 5]
```diff
@@ -0,0 +1,36 @@
+import { AppPipelineType } from "./types";
+import { getCookie, PipeLineHostToApi } from "./utils";
+import { API_ROUTE } from "..";
+
+export const requestCreatePipeline = async (pipelineData: AppPipelineType, creation: boolean): Promise<boolean> => {
+	const jwt = getCookie("aeris_jwt");
+
+	const request = API_ROUTE + "/workflow/" + (creation ? pipelineData.id : "")
```
Comment (AnonymusRaccoon) [inline web-app/src/utils/CRUDPipeline.tsx:8]: Ternary failed

**Review by AnonymusRaccoon — APPROVED** [review]

**Pull request merged**